packages/devextreme/js/__internal/ui/splitter/resize_handle.ts

```diff
@@ -45,12 +55,42 @@ class ResizeHandle extends (Widget as any) {
   _initMarkup(): void {
     super._initMarkup();
 
-    this._toggleDirectionClass();
-    this.$element().addClass(RESIZE_HANDLE_CLASS);
+    this._renderResizeHandleContent();
 
     this._setAriaAttributes();
   }
 
+  _renderResizeHandleContent(): void {
+    this.$element().addClass(RESIZE_HANDLE_CLASS);
+    this._toggleDirectionClass();
+
+    this._$collapsePrevButton = $('<div>').addClass(this._getCollapseIconClass(false)).appendTo(this.$element());
+    this._$resizeHandle = $('<div>').addClass(RESIZE_HANDLE_ICON_CLASS).appendTo(this.$element());
+    this._$collapseNextButton = $('<div>').addClass(this._getCollapseIconClass(true)).appendTo(this.$element());
+
+    this._setResizeHandleContentVisibility();
+  }
+
+  _getCollapseIconClass(isNextButton: boolean): string {
+    const isHorizontal = this._isHorizontalDirection();
+    let classList = `${isNextButton ? RESIZE_HANDLE_COLLAPSE_NEXT_BUTTON_CLASS : RESIZE_HANDLE_COLLAPSE_PREV_BUTTON_CLASS} ${ICON_CLASS}`;
+    if (isNextButton) {
+      classList += ` dx-icon-spin${isHorizontal ? 'right' : 'down'}`;
+    } else {
+      classList += ` dx-icon-spin${isHorizontal ? 'left' : 'up'}`;
+    }
+
+    return classList;
+  }
+
+  _setResizeHandleContentVisibility(): void {
+    const { showCollapsePrev, showCollapseNext, resizable } = this.option();
+
+    this._$collapsePrevButton.toggleClass(RESIZE_HANDLE_HIDDEN_BUTTON, !showCollapsePrev);
```

Comment:
We should use display: none or 'dx-state-invisible' class to hide a control elements

Reply:
done